### DISCUSSION TITLE
"mysqldump is missing" error when trying to update core

### DISCUSSION THREAD

**Discussion by aharonach** [atrocore/atrocore#666] [Q&A] [answered]
Hi,
I have just installed AtroPIM + AtroCore on an AWS EC2 instance connected to an RDS database.
The system has an update but fails when I try to do so because it can't find "mysqldump" in the server to do a backup before the update:

```
Loading composer repositories with package information
Creating restore point
Failed! Please, install mysqldump! System can't create dump for database!
```

Although there is a mysqldump installed on the server.

Can someone walk me through this?

**Comment by rratsun** [comment, marked as answer]
Try to call ```mysqldump -h {$config['database']['host']} -u {$config['database']['user']} --port={$port} -p'{$config['database']['password']}' {$config['database']['dbname']} > dump.sql``` manually to understand the problem. Our updater call this command to create dump, so in case of errors update will fault.

**Reply by aharonach** [reply]
Thanks!
The problem was the setting "column-statistics" which was on by default. I disabled it and then it worked.
